### PR TITLE
patch: Publish release to WinGet (Windows Package Manager)

### PR DESCRIPTION
**Pull request by vedantmgoyal2009**
This action automatically generates manifests for WinGet Community Repository ([microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs)) and submits them.

Before merging this:
1. Please add a GitHub token with `public_repo` scope as a repository secret and rename the secret name in the workflow.
2. Fork [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs) under @balena-io.

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
If anyone can review and merge it please, it will help in keeping the package at winget updated.

@dfunckt @chrisys @thgreasi @tmigone @iamsolankiamit @Bucknalla @iayanpahwa @LucianBuzzo @vipulgupta2048

**Comment by lurch** [comment]
Perhaps the Balena employees don't work over the weekend? :thinking:

**Comment by vipulgupta2048** [comment]
Thanks for opening the PR, you seem to have missed the commit prefix to signify the change-type for your PR. Refer to this [contributing guide](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#creating-commits-in-line-with-semantic-versioning] and let us know if you have any further questions. Also, mentioned [here](https://github.com/balena-io/etcher/blob/master/docs/COMMIT-GUIDELINES.md#change-type-type).

**Comment by vipulgupta2048** [comment]
> Perhaps the Balena employees don't work over the weekend? thinking

@lurch most of them don't but It's a Monday somewhere in the world 💀

**Comment by vedantmgoyal2009** [comment]
@vipulgupta2048 I've fixed the commit message.

**Comment by thgreasi** [comment]
@vipulgupta2048 let's arch call this before merging

**Comment by vedantmgoyal2009** [comment]
What is arch call?

**Comment by mcraa** [comment]
@vedantmgoyal2009 
arch call is where we discuss things internally.

you changed the [commit message](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#creating-commits-in-line-with-semantic-versioning) but did not fix it. You may do something like
```
patch: publish to winget with gh action
```

**Comment by vedantmgoyal2009** [comment]
Hi, I have changed the commit message. BTW, you can use squash and merge to provide the commit message of your choice.

**Comment by mcraa** [comment]
hi @vedantmgoyal2009
the status is, that you did not fix the commit message, which blocks the pre-merge checks.
We plan to merge it before the next release.
We want to add other github actions as well, probably we will change this PR to another branch than `master`
so the pre-merge checks won't block it and merge later to master together with other changes.

**Comment by mcraa** [comment]
@vedantmgoyal2009 sorry, I missed that, the checks did not re-run somehow.
We will proceed as I said earlier.

**Comment by vedantmgoyal2009** [comment]
Any updates?

**Comment by mcraa** [comment]
There is no work in progress that could be released quickly.
In parallel, we are updating the CI,
most probably the next release will come through the new CI already 
(and with this gh action to winget)

**Comment by mcraa** [comment]
@balena-ci retest!

**Comment by mcraa** [comment]
@balena-ci test

File: .github/workflows/winget.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Publish to WinGet
+on:
+  release:
+    types: [released]
+jobs:
+  publish:
+    runs-on: windows-latest # action can only be run on windows
+    steps:
+      - uses: vedantmgoyal2009/winget-releaser@latest
```
Comment (thgreasi) [inline .github/workflows/winget.yml:9]: @mcraa should we pin this to a specific commit as discussed in the arch call?

Comment (mcraa) [reply .github/workflows/winget.yml:9]: will do in #3780

**Pull request merged**